class.two-factor-core.php

```diff
@@ -58,7 +58,11 @@ public static function add_hooks() {
 	 * Sites on WordPress 4.6+ benefit from just-in-time loading of translations.
 	 */
 	public static function load_textdomain() {
-		load_plugin_textdomain( 'two-factor' );
+		load_plugin_textdomain(
+			'two-factor',
+			false,
+			dirname(plugin_basename(__FILE__)).'/languages/'
```

Comment:
The plugin is using the translations provided by WordPress.org so we don't bundle them with the plugin.

Reply:
@Sonic853 This also needs to be removes since WP core now handles the translation files.

Reply:
Updated.